akka-http-core/src/main/java/akka/http/javadsl/model/headers/XForwardedHost.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model.headers;
+
+/**
+ *  Model for the `X-Forwarded-Host` header.
+ *  Specification: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host
+ */
+public abstract class XForwardedHost extends akka.http.scaladsl.model.HttpHeader {
+    public abstract akka.http.javadsl.model.Host getAddress();
```

Comment:
I suggest to rename this method to `getHost`. The type is `Host` and the Scala case class member is also called `host`.